pkg/addons/applier.go

```diff
@@ -150,19 +151,13 @@ func (a *Applier) GenerateHelmConfigsForRestore() ([]v1beta1.Chart, []v1beta1.Re
 	return charts, repositories, nil
 }
 
+// GetAirgapCharts is deprecated in favor of GetBuiltinCharts.
 func (a *Applier) GetAirgapCharts() ([]v1beta1.Chart, []v1beta1.Repository, error) {
```

Comment:
you can totally remove this function if you want
its only use should be in the version metadata command, no?